Content.Client/Chemistry/UI/ReagentDispenserWindow.xaml.cs

```diff
@@ -86,6 +85,11 @@ public void UpdateState(BoundUserInterfaceState state)
             UpdateContainerInfo(castState);
             UpdateReagentsList(castState.Inventory);
 
+            if (castState.OutputContainerEntity is { } containerEntity)
+                View.SetEntity(containerEntity);
+            else
+                View.SetEntity(null);
```

Comment:
```suggestion
            View.SetEntity(castState.OutputContainerEntity);
```
nullable args moment

Reply:
Weird. There's SetEntity(EntityUid? uid) and SetEntity(NetEntity netEnt). But not SetEntity(NetEntity? netEnt).

Am I gonna make an engine PR for a single question mark? Possibly.

Reply:
Nope, I'm not, since that makes a bunch of other SetEntity calls ambiguous.

Reply:
So we'll TryGetEntity the netent first, then use that nullable as the argument to SetEntity. Means we need to add entity manager as a dependency, but it works!